planetscale/certs.go

```diff
@@ -110,10 +118,15 @@ func (c *certificatesService) Create(ctx context.Context, r *CreateCertificateRe
 		return nil, fmt.Errorf("parsing certificate chain failed: %s", err)
 	}
 
+	privateKeyBytes, err := x509.MarshalPKCS8PrivateKey(r.PrivateKey)
```

Comment:
This format is more generic and then encodes the key type itself so we can use it properly.